docs/pouch-runv-guide.md

```diff
@@ -58,3 +64,36 @@ cd hyperstart
 ./configure
 sudo make
 ```
+
+Copy guest kernel and image to the default directory which runv will look for.
+```
+mkdir /var/lib/hyper/
+cp build/{kernel,hyper-initrd.img} /var/lib/hyper/
+```
+
+## Get Started
+
+supported containerd commit `31dabf0c7d321e14f49202e8016aa6b5a0905bc0`
```

Comment:
Does runv running support containerd 1.0.0?
If yes, maybe changing this commit ID to 1.0.0 could be better?

Reply:
it will be next step.